static/js/ktl-component/why-teach/style.scss

```diff
@@ -0,0 +1,49 @@
+.why-teach-grid {
+  &__nav {
+    display: none;
+  }
+  @media (min-width: 1200px) {
+    display: flex;
+    &__content {
+      max-width: 70%;
+    }
+    &__nav {
+      display: block;
+      margin-left: 96px;
+    }
+  }
+}
+
+.why-teach-nav {
+  position: sticky;
+  top: 96px;
+}
+
+.quote-section {
+  &__info {
+    display: none;
+  }
+  @media (min-width: 768px) {
+    &__grid {
+      display: flex;
+    }
+    &__content {
+      max-width: 70%;
+    }
+    &__info {
+      display: block;
+      margin-left: 32px;
+    }
+  }
+}
+
+.ktl-with-anchor {
```

Comment:
What does `ktl-with-anchor` do? if it's should have anchor link, i haven't seen it.

Reply:
Actually it is a good case to show link id as on main. May be we will create task for that i component?

Reply:
It's compensation for sticky header